articles/best-practices/rules.md

```diff
@@ -233,7 +233,7 @@ Updating a user via use of the `auth0` object ultimately results in a call to th
 
 The [`context`](#context-object) object contains the `primaryUser` property which refers to the user identifier of the primary user. This user identifier will typically be the same as `user_id` property in the root of the `user` object. The primary user is the user that is returned to the Auth0 engine when the rule pipeline completes, and the `user_id` is a unique value generated by Auth0 to uniquely identify the user within the Auth0 tenant. This `user_id` should be treated as an opaque value.
 
-There are occasions when `primaryUser` must be updated as the primary user may change&mdash;i.e., the user returned to the Auth0 engine will be different from the user on rule pipeline entry; [automatic account linking](/users/concepts/overview-user-account-linking#automatic-account-linking) is the primary use case. On such occasions, a rule must update `primaryUser` to reflect the new primary user identifier. Note that this change *will not* affect any subsequent rule executed in the current instance of the pipeline; the `user` object will remain unchanged.
+There are occasions when `primaryUser` must be updated as the primary user may change&mdash;i.e., the user returned to the Auth0 engine will be different from the user on rule pipeline entry. On such occasions, a rule must update `primaryUser` to reflect the new primary user identifier. Note that this change *will not* affect any subsequent rule executed in the current instance of the pipeline; the `user` object will remain unchanged.
```

Comment:
@aaguiarz what are other cases where the user returned to the Auth0 engine will be different from the user on rule pipeline entry? Does this sentence still make sense if we remove the account linking?

Reply:
@esarafianou No, this only applies to account linking. However, we are not removing account linking, we are changing the guidance on how to trigger it.